### ISSUE TITLE
Support environment variable configuration for Docker

### ISSUE THREAD

**Issue by chauffer** [Kwoth/NadekoBot#2555]
Hey! Would be great if NadekoBot supported environment variables for Docker as opposed to files for configuration.

Using NadekoBot on an orchestrator such as Rancher will heavily simplify operational work needed.

**Comment by Kwoth** [comment]
You mean for credentials? It's already supported.

**Comment by chauffer** [comment]
Yep, for credentials. I couldn't find it in the docs by looking for `environment`. Looking at the code I found https://github.com/Kwoth/NadekoBot/blob/7ee041b5d9526a2a3e22406938f38effa2316325/NadekoBot.Core/Services/Impl/BotCredentials.cs#L55 which is probably where it''s happening, but I could not find any example.

**Comment by LordVeovis** [comment]
@chauffer Just declare the credentials parameters prefixed with NadekoBot_ like below:

docker run --rm -it -e NadekoBot_ClientId=1234 -e NadekoBot_Token=MzI4eraera -e TotalShars=1 mynadeko
